storage/innobase/xtrabackup/doc/source/release-notes/8.0/8.0.27-19.0.rst

```diff
@@ -0,0 +1,49 @@
+.. _PXB-8.0.NEXT:
+
+================================================================================
+*Percona XtraBackup* 8.0.NEXT
+================================================================================
+
+:Date: January  2022
+:Installation: `Installing Percona XtraBackup <https://www.percona.com/doc/percona-xtrabackup/8.0/installation.html>`_
+
+Percona XtraBackup enables MySQL backups without blocking user queries, making it ideal for companies with large data sets and mission-critical applications that cannot tolerate long periods of downtime. Offered free as an open source solution, it drives down backup costs while providing unique features for MySQL backups.
+
+Release Highlights
+=================================================
+
+The following list are some of the bug fixes for *MySQL* 8.0.27, provided by Oracle, and included in Percona Server for MySQL:
```

Comment:
The following list **are** -> The following list **contains**?

Reply:
Rewritten.